client/src/components/CreateCampaignStep2.js

```diff
@@ -1,63 +1,248 @@
+              <option value="MN">Minnesota</option>
+              <option value="MS">Mississippi</option>
+              <option value="MO">Missouri</option>
+              <option value="MT">Montana</option>
+              <option value="NE">Nebraska</option>
+              <option value="NV">Nevada</option>
+              <option value="NH">New Hampshire</option>
+              <option value="NJ">New Jersey</option>
+              <option value="NM">New Mexico</option>
+              <option value="NY">New York</option>
+              <option value="NC">North Carolina</option>
+              <option value="ND">North Dakota</option>
+              <option value="OH">Ohio</option>
+              <option value="OK">Oklahoma</option>
+              <option value="OR">Oregon</option>
+              <option value="PA">Pennsylvania</option>
+              <option value="RI">Rhode Island</option>
+              <option value="SC">South Carolina</option>
+              <option value="SD">South Dakota</option>
+              <option value="TN">Tennessee</option>
+              <option value="TX">Texas</option>
+              <option value="UT">Utah</option>
+              <option value="VT">Vermont</option>
+              <option value="VA">Virginia</option>
+              <option value="WA">Washington</option>
+              <option value="WV">West Virginia</option>
+              <option value="WI">Wisconsin</option>
+              <option value="WY">Wyoming</option>
+            </FormControl>
+          </Col>
+          <Col xs={3}>
+            <ControlLabel>ZIP</ControlLabel>
+            <FormControl type="number" name="zip" />
+          </Col>
+        </FormGroup>
+        <FormGroup>
+          <h2>Waste Collection Provider:</h2>
+          <Col xs={12}>
+            <ControlLabel>CURRENT PROVIDER</ControlLabel>
+            <FormControl
+              componentClass="select"
+              name="wasteMgmtName"
+              onChange={this.handleWasteMgmtChange}
+            >
+              {this.renderWasteProviderOptions(wasteProviders)}
+            </FormControl>
+            <ControlLabel>PHONE</ControlLabel>
+            <FormControl
+              className="readonly"
+              type="tel"
+              name="wasteMgmtPhone"
+              value={this.state.activeProvider.phone}
+              readOnly
+            />
+            <ControlLabel>EMAIL</ControlLabel>
+            <FormControl
+              className="readonly"
+              type="email"
+              name="wasteMgmtEmail"
+              value={this.state.activeProvider.email}
+              readOnly
+            />
+          </Col>
+        </FormGroup>
+
+        <FormGroup>
+          <h2>Building Info:</h2>
+          <Col xs={12}>
+            <ControlLabel>THIS PROPERTY CONSISTS OF:</ControlLabel>
+          </Col>
+          <Col xs={2}>
+            <FormControl type="Number" name="buildingCount" placeholder={0} />
```

Comment:
disable out of range numbers ie 0 < x < 100

Reply:
ok